docs/SUMMARY.md

```diff
@@ -1,13 +1,6 @@
 # Summary
 
 * [Welcome](README.md)
-    * [Getting started](getting_started/README.md)
-        * [An HTTP Server](getting_started/http_server.md)
-        * [A Command Line Application](getting_started/cli.md)
-    * [Using the compiler](using_the_compiler/README.md)
-    * [The shards command](the_shards_command/README.md)
-    * [Coding style](conventions/coding_style.md)
-    * [Platform Support](platform_support.md)
```

Comment:
> The following pages exist in the docs directory, but are not included in the "nav" configuration:
>   - platform_support.md

Reply:
I think we can probably put this in the specification tree?

Reply:
I don't really see it fitting in the description of the language. The closest I can think of is to put it as part of the manual of the compiler.

Reply:
Ah I see that we have already stuff related to the compiler in that section, so OK